src/docs/marble/svg/export-to-png.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+echo "### Removing all older pngs ###";
```

Comment:
this is unnecessary as the `inkscape` export will replace existing files without warning. It is even a bit dangerous, as if `inkscape` is not installed this script would just delete all PNGs.

Reply:
One idea I had is to begin the script with an `inkscape -zV` (printing the version, no UI). This way if the command is unknown the script will stop

Reply:
```sh
#!/bin/sh
inkscape=$(which inkscape)
if [ -x "$inkscape" ] ; then
  echo "### Removing all older pngs ###";
  for f in "../png/"*.png ;
  do
   rm "$f";
  done
  echo "### Cleared old pngs ###";

  echo "### Converting SVG -> PNG ###";
  for f in *.svg ;
  do
   inkscape "$f" --export-png="../png/`basename $f .svg`.png" --without-gui;
  done
  echo "### Conversion complete ###";
else
  echo "inkscape executable could not be found in the PATH. Please add it to PATH variable and retry"
fi
```

Reply:
> this is unnecessary as the `inkscape` export will replace existing files without warning. It is maybe even a bit dangerous, as if `inkscape` is not installed this script would just delete all PNGs.

Reason for having the cleanup part is to make sure that pngs are always in sync with svgs. Otherwise when we rename, we need to remember to delete corresponding existing png

After a rename if we forget to updated javadoc, users will complain that the image is not showing. Else, it would continue to show the old png

Reply:
got it, you convinced me 👍
the `-x inkscape` should be good too